instance/templates/instance/ansible/vars.yml

```diff
@@ -26,19 +31,11 @@ CMS_HOSTNAME: '{{ instance.studio_domain_nginx_regex }}'
 EDXAPP_SESSION_COOKIE_DOMAIN: ''
 
 # Nginx
-{% if instance.protocol == 'https' %}
-NGINX_ENABLE_SSL: true
-NGINX_REDIRECT_TO_HTTPS: true
-{% else %}
+NGINX_SET_X_FORWARDED_HEADERS: false
+
+# SSL is handled on the load balancer, and the appservers are HTTP only.
 NGINX_ENABLE_SSL: false
```

Comment:
@bradenmacdonald I tried this on a Ficus test instance, and apparently this still does not work, see https://console.opencraft.com/instance/2520/edx-appserver/947/.  We would have to either fix configuration to provide snakeoil certificates if no certificate is provided, or generate our own snakeoil certs and pass them in (the former probably makes more sense).  Can we exclude that from the current scope?  Do you want me to create a separate ticket for this?